families/seth/src/sawtooth_seth/processor/handler/sawtooth_app_state.go

```diff
@@ -44,7 +44,8 @@ func NewSawtoothAppState(state *processor.Context) *SawtoothAppState {
 // GetAccount retrieves an existing account with the given address. Returns nil
 // if the account doesn't exist.
 func (s *SawtoothAppState) GetAccount(addr Word256) *Account {
-	vmAddress, err := NewEvmAddrFromBytes(addr.Bytes()[:20])
+	addrBytes := addr.Bytes()[len(addr.Bytes())-20:]
```

Comment:
`len(addr.Bytes())` is constant, so you can just use `Word256Length`: https://github.com/hyperledger/sawtooth-core/blob/master/families/seth/src/burrow/word256/word.go#L32

`vmAddress, err := NewEvmAddrFromBytes(addr.Bytes()[Word256Length-20:]`